### PR TITLE
TRT-1627: Use variant registry for disruption and remove some outdated half implemented view management

### PR DESCRIPTION
**Pull request by dgoodwin**
Disruption stack is ported to use a new view, JobsWithVariants, instead of using it's own copy of the logic to determine variants. Most of the work to switch to the registry happened on the views + scheduled queries side, this change is predominantly switching to use that data instead of the old. A followup PR will attempt to do more, and stop uploading things we no longer need.

Removed the code creating various views, they were long out of date, and only a handful of views had schema management. Would be nice to have a proper solution for this but this wasn't sufficient for our needs today, and I don't want two copies of that schema floating around out of date.

### CODE REVIEW

**Comment by dgoodwin** [comment]
/assign @xueqzhan 

Ken could you have a look at this first round of changes?

**Review by xueqzhan — COMMENTED** [review]
Other than the lint error, the change looks good to me.

File: pkg/jobrunaggregator/jobrunaggregatorapi/types_row_job.go [hunk 1]
```diff
@@ -105,14 +47,19 @@ type JobRow struct {
...
+	GCSBucketName               string
+	GCSJobHistoryLocationPrefix string
+	CollectDisruption           bool
+	CollectTestRuns             bool
 	Platform                    string
 	Architecture                string
 	Network                     string
 	IPMode                      string
 	Topology                    string
 	Release                     string
```
[the remark this answers is not in the document]
Comment (dgoodwin) [reply pkg/jobrunaggregator/jobrunaggregatorapi/types_row_job.go:63]: FromRelease is actually not set on jobs that do not upgrade, and comes out null here as a result.

File: pkg/jobrunaggregator/jobrunaggregatorapi/types_row_job.go [hunk 2]
```diff
@@ -34,66 +36,6 @@ const (
     "description": "should we collect test run data from the job runs",
     "type": "BOOLEAN",
     "mode": "REQUIRED"
-  },
```
[the remark this answers is not in the document]
Comment (dgoodwin) [reply pkg/jobrunaggregator/jobrunaggregatorapi/types_row_job.go:37]: Ah good point, I missed that one. Found several others while removing it.

**Pull request merged**